api/v1/webspherelibertyapplication_types.go

```diff
@@ -850,6 +865,21 @@ func (s *WebSphereLibertyApplicationService) GetBindable() *bool {
 	return s.Bindable
 }
 
+func (np *WebSphereLibertyApplicationNetworkPolicy) GetFromLabels() map[string]string {
+	if np == nil {
+		return nil
+	}
+	return np.FromLabels
+}
+
+func (np *WebSphereLibertyApplicationNetworkPolicy) IsNotDefined() bool {
+	return np == nil
+}
+
+func (np *WebSphereLibertyApplicationNetworkPolicy) IsEmpty() bool {
+	return np != nil && (np.FromLabels == nil || len(np.FromLabels) == 0)
+}
+
```

Comment:
Add {NetworkPolicy,v1} to list of resources

add common.GetComponentNameLabel(cr): cr.Name to GetLabels method